ert_gui/tools/file/file_dialog.py

```diff
@@ -129,3 +144,7 @@ def _append_text(self, text: str) -> None:
         if self._follow_mode:
             self._view.moveCursor(QTextCursor.End)
         self._view.appendPlainText(text)
+        self.adjustSize()
+
+    def sizeHint(self) -> QSize:
```

Comment:
does this have to be camelCased?

Reply:
it looks like that's the convention of the python qt stuff